### PR TITLE
chores(medusa-core-utils): export create medusa container from utils

### PR DESCRIPTION
**Pull request by adrien2p**
export `createMedusaContainer` from utils

### CODE REVIEW

File: packages/medusa-core-utils/src/index.ts [hunk 1]
```diff
@@ -12,3 +12,4 @@ export * from "./medusa-container"
 export { parseCorsOrigins } from "./parse-cors-origins"
 export { transformIdableFields } from "./transform-idable-fields"
 export { default as zeroDecimalCurrencies } from "./zero-decimal-currencies"
+export { createMedusaContainer } from "./medusa-container"
```
Comment (olivermrbl) [inline packages/medusa-core-utils/src/index.ts:15]: thought(non-blocking): Should we move it to `@medusajs/utils` while we are at it? 😜

**Review by carlos-r-l-rodrigues — APPROVED** [review]

File: packages/medusa-core-utils/src/index.ts (hunk 1 above)
Comment (olivermrbl) [reply packages/medusa-core-utils/src/index.ts:15]: @adrien2p We can also merge this now, if it is blocking and then do the move in a subsequent PR. What do you think?

Comment (carlos-r-l-rodrigues) [reply packages/medusa-core-utils/src/index.ts:15]: at line 11 we have `export * from "./medusa-container"`

Comment (adrien2p) [reply packages/medusa-core-utils/src/index.ts]: No it is exported in fact, it was just my IDE playing with me 😂

Comment (adrien2p) [reply packages/medusa-core-utils/src/index.ts]: So I just do the move to utils/types only

File: packages/utils/src/index.ts [hunk 2]
```diff
@@ -1 +1,2 @@
 export * from "./decorators"
+export * as ContainerUtils from "./container"
```
Comment (carlos-r-l-rodrigues) [inline packages/utils/src/index.ts:2]: Would make sense that this is a `CommonTypes`? `common.ts`, as well as `CommonUtils`
Container is not a "domain"

Comment (adrien2p) [reply packages/utils/src/index.ts:2]: Yes indeed, I could place it in common 👍 I ll do that

Comment (adrien2p) [reply packages/utils/src/index.ts]: done

**Review by olivermrbl — COMMENTED** [review]
LGTM - is it ready for review? 😄

**Comment by olivermrbl** [comment]
Remember the changeset :)

**Comment by adrien2p** [comment]
> Remember the changeset :)

Yes it should be, just testing locally. Also, I will create the changeset now that I ve touch all the packages 😂

**Comment by adrien2p** [comment]
Screenshot 2023-03-17 at 15 08 58

I still have an error, @carlos-r-l-rodrigues Isn't the stock location optional?

**Comment by carlos-r-l-rodrigues** [comment]
> I still have an error, @carlos-r-l-rodrigues Isn't the stock location optional? In that case the product variant inventory should not destructure the container as it will resolve it

It is optional. And it seems to be something in your local.

**Pull request closed without merging**